bears/vcs/git/GitCommitBear.py

```diff
@@ -102,7 +89,6 @@ def check_shortlog(self,
                                          care").
         :param shortlog_wip_check:       Whether a wip in the shortlog should
                                          yield a major result or not.
```

Comment:
Could you change the description to
Whether a WIP in the shortlog text should yield a major result or not.